lib/response.js

```diff
@@ -326,6 +326,9 @@ res.jsonp = function jsonp(obj) {
  */
 
 res.sendStatus = function sendStatus(statusCode) {
+  if (typeof statusCode !== 'number') {
+    throw new TypeError('Invalid status code: ' + statusCode);
```

Comment:
```suggestion
    throw new TypeError(`statusCode must be a valid number to res.sendStatus`);
```

I think it seems consistent

Reply:
agree